File: Python/2021/1stOct_rutvikgondaliya.py
def prime_checker(n):
    if n > 1:
        for k in range(2, n // 2 + 1):
            if (n % k) == 0:
                return False
        else:
            return True
    else:
        return False

num = int(input("Enter number of Input: "))

num1, num2 = 1, 1 
count = 0

if num == 1:
    print("Fibonacci series: ")
    print(num1)
else:
    print("Fibonacci series: ")
    while count < num:
        if not prime_checker(num1) and num1 % 5 != 0:
            print(num1, end=' ')
        else:
            print(0, end=' ')
        sum = num1 + num2
        num1 = num2
        num2 = sum
        count = count + 1

File: JAVA/2021/1stOct_rutvikgondaliya.java
package java_class;
import java.util.*;
public class main {   
	 public static boolean isPrimeChecker(int no)
	    {
	        // checking Corner case
	        if (no <= 1)
	            return false;
	  
	        // Check from 2 to n-1
	        for (int j = 2; j < no; j++)
	            if (no % j == 0)
	                return false;
	  
	        return true;
	    }
	  
	    public static void main(String[] args) {
	      Scanner scanner = new Scanner(System.in);
	      int num = scanner.nextInt();
	      int num1 = 1;
	      int num2 = 1;
	      System.out.print(num1+" "+num2+" ");
	      for(int k=2; k < num; k++)
	      {
	        int sum = num1 + num2;  
	        if(sum % 5 == 0)
	             System.out.print(0+" ");
	        else if(isPrimeChecker(sum))
	             System.out.print(0+" ");
	        else
	             System.out.print(sum+" ");
	        num1=num2;
	        num2=sum;
	      }
	    }
	}
